src/main/java/ReadManager.java

```diff
@@ -0,0 +1,25 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadManager {
```

Comment:
Can you use the default access modifier?

Reply:
fixed